addons/l10n_ua/data/template/account.tax-ua_ias.csv

```diff
@@ -1,81 +1,74 @@
+"","","","","","","","","","","tax","refund","","",""
+"purchase_tax_template_vat20","19","20%","","20%","20.0","purchase","tax_group_vat20","","","base","invoice","","Придбання з ПДВ 20%","+ ПДВ 20%"
+"","","","","","","","","","","tax","invoice","ua_ias_1140","",""
+"","","","","","","","","","","base","refund","","",""
+"","","","","","","","","","","tax","refund","ua_ias_1140","",""
+"purchase_tax_template_vat20incl","19","20% INC","","20% INC","20.0","purchase","tax_group_vat20","False","True","base","invoice","","Придбання в т. ч. ПДВ 20%","в т. ч. ПДВ 20%"
+"","","","","","","","","","","tax","invoice","ua_ias_1140","",""
+"","","","","","","","","","","base","refund","","",""
+"","","","","","","","","","","tax","refund","ua_ias_1140","",""
+"purchase_tax_template_vat14","20","14%","","14%","14.0","purchase","tax_group_vat14","","","base","invoice","","Придбання з ПДВ 14%","+ ПДВ 14%"
+"","","","","","","","","","","tax","invoice","ua_ias_1140","",""
+"","","","","","","","","","","base","refund","","",""
+"","","","","","","","","","","tax","refund","ua_ias_1140","",""
+"purchase_tax_template_vat14incl","20","14% INC","","14% INC","14.0","purchase","tax_group_vat14","False","True","base","invoice","","Придбання в т. ч. ПДВ 14%","в т. ч. ПДВ 14%"
+"","","","","","","","","","","tax","invoice","ua_ias_1140","",""
+"","","","","","","","","","","base","refund","","",""
+"","","","","","","","","","","tax","refund","ua_ias_1140","",""
+"purchase_tax_template_vat7","21","7%","","7%","7.0","purchase","tax_group_vat7","","","base","invoice","","Придбання з ПДВ 7%","+ ПДВ 7%"
+"","","","","","","","","","","tax","invoice","ua_ias_1140","",""
+"","","","","","","","","","","base","refund","","",""
+"","","","","","","","","","","tax","refund","ua_ias_1140","",""
+"purchase_tax_template_vat7incl","21","7% INC","","7% INC","7.0","purchase","tax_group_vat7","False","True","base","invoice","","Придбання в т. ч. ПДВ 7%","в т. ч. ПДВ 7%"
+"","","","","","","","","","","tax","invoice","ua_ias_1140","",""
+"","","","","","","","","","","base","refund","","",""
+"","","","","","","","","","","tax","refund","ua_ias_1140","",""
+"purchase_tax_template_vat0","22","0%","","0%","0.0","purchase","tax_group_vat0","","","base","invoice","","Придбання з ПДВ 0%","ПДВ 0%"
+"","","","","","","","","","","tax","invoice","","",""
+"","","","","","","","","","","base","refund","","",""
+"","","","","","","","","","","tax","refund","","",""
+"purchase_tax_template_vat_free","23","VAT Exempt","","VAT Exempt","0.0","purchase","tax_group_vat0","","","base","invoice","","Придбання звільнене від ПДВ","Звільнено від ПДВ"
+"","","","","","","","","","","tax","invoice","","",""
+"","","","","","","","","","","base","refund","","",""
+"","","","","","","","","","","tax","refund","","",""
+"purchase_tax_template_vat_not","24","VAT NS","","VAT NS","0.0","purchase","tax_group_vat0","","","base","invoice","","Придбання не є об'єктом ПДВ","Не є об'єктом ПДВ"
```

Comment:
Name should be 0% NS, Why NS ?

Reply:
Shorthand for "Not Subject". It comes from l10n_es. You can see an example here: https://github.com/odoo/odoo/blob/a773c1d289e74a788bab2049fee226a3eb2db430/addons/l10n_es/data/template/account.tax-es_common.csv#L124

If "NS" isn't clear, would it be better to write it as "0% Not Subject" instead?